src/status_im2/navigation/screens.cljs

```diff
@@ -112,13 +112,11 @@
      :component add-new-contact/new-contact}
 
     {:name      :how-to-pair
-     :options   {:theme  :dark
```

Comment:
I noticed these screens phone status bar (time/battery etc) was in the wrong color for the theme